pythonforandroid/bootstraps/sdl2/build/src/patches/SDLActivity.java.patch

```diff
@@ -1,9 +1,18 @@
 --- a/src/main/java/org/libsdl/app/SDLActivity.java
 +++ b/src/main/java/org/libsdl/app/SDLActivity.java
-@@ -196,6 +196,15 @@ public class SDLActivity extends Activity implements View.OnSystemUiVisibilityCh
+@@ -94,6 +94,8 @@
+     // This is what SDL runs in. It invokes SDL_main(), eventually
+     protected static Thread mSDLThread;
+
++    public static int keyboardInputType = InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD;
```

Comment:
So at the end the changes you made to this `SDLActivity.java.patch` file is only using this const and clearing some trailing whitespaces, correct or I'm missing something?

Reply:
You are correct.